### PR TITLE
Add raw data for 2012 winners

### PR DESCRIPTION
**Pull request by staaas**
I think we can benefit from committing data separately from migrations. I didn't have enough time to finish the migration, so I'm committing everything I've finished so far.

### CODE REVIEW

**Comment by abitrolly** [comment]
How about moving data into `/data` and add README.md describing how it is used and how to use it? This is essentially an open data project. It will also allow us to mature data/api tree structure for subsequent access.

**Comment by abitrolly** [comment]
I also don't like that people names are just names - there could be data loss if names match. Is it possible to detect duplicates and export ids as well?

**Comment by staaas** [comment]
@abitrolly sounds good to me

let's merge this and fix/add your suggestions in subsequent prs/sprints

**Comment by abitrolly** [comment]
LGTM. Don't you mind moving it into https://github.com/kinaklub/next.filmfest.by/tree/master/data/raw ?

File: results/migrations/raw_data/2012_winners/data.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
+[
+    {
+        "film_title_en": "The Great Race",
```
Comment (davojta) [inline results/migrations/raw_data/2012_winners/data.json:3]: i think we need to calculate id to films based on lower case name + year
or id it to exist films data base with stub data

i will open separate ticket for that for future sprints

**Comment by davojta** [comment]
@abitrolly  is that ticket https://github.com/kinaklub/next.filmfest.by/issues/45 what you are talking about?

**Comment by davojta** [comment]
https://github.com/kinaklub/next.filmfest.by/issues/46 -оne more ticket for future sprint

**Comment by abitrolly** [comment]
@davojta it is better to avoid too many file moves, because it complicates history navigation.

**Comment by davojta** [comment]
@abitrolly  so need to fix it here )

**Comment by davojta** [comment]
@nott it is your private branch, can you finish it by yourself ?

**Comment by davojta** [comment]
this should be close as work have been moved to #56

**Comment by davojta** [comment]
close

**Pull request closed without merging**